collections/github-browser-extensions/index.md

```diff
@@ -15,6 +15,7 @@ items:
  - ProLoser/Github-Omnibox
  - Justineo/github-hovercard
  - panzerdp/clipboardy
+ - rgehan/octolenses-browser-extension
```

Comment:
Please add this to the end of the list rather than the middle; it's still pretty new. Thanks!